src/components/OneContainerLayout/OneContainerLayout.tsx

```diff
@@ -0,0 +1,49 @@
+import { PropsWithChildren } from 'react';
+import styled from 'styled-components';
+import React from 'react';
+
+/** Layout for one container on page, like on mentor page */
+
+type OneContainerProps = React.PropsWithChildren<{
+  headLine: string;
+  children: React.ReactNode;
+}>;
+
+// eslint-disable-next-line react/prop-types
+const OneContainerLayout: React.FC<PropsWithChildren<OneContainerProps>> = ({
```

Comment:
I had some trouble rendering the children, and this is what I came up with while trying to learn the syntax. I'm not sure if I am able to fix this...

Reply:
Ok,

I mean it should just work like this

```jsx
type Props = {
foo: string
}

const Bar: React.FC<Props> = ({foo, children}) => (<div><span{foo}</span> and also {children}</div>)
```

because children are included in React.FC automatically